weave

```diff
@@ -1747,6 +1747,7 @@ launch_plugin_if_not_running() {
     if ! PLUGIN_CONTAINER=$(docker run -d --name=$PLUGIN_CONTAINER_NAME \
         $(docker_run_options) \
         $RESTART_POLICY \
+        --pid=host \
```

Comment:
If we don't do this we cannot navigate to the container's namespace given its PID.  At least I don't know how to do it.